lib/OpenQA/Schema/Result/JobGroups.pm

```diff
@@ -172,7 +173,9 @@ sub matches_nested {
 }
 
 # check the group comments for important builds
-sub _important_builds ($self) {
+sub important_builds ($self) {
+    if (my $cached = $self->{_important_builds}) { return $cached }
```

Comment:
Should this be invalidated somewhere? At least I don't see it. Or do we assume they don't change after the fact?

Reply:
These result objects are never being invalidated automatically. One has to call discard_changes manually for that. I don't think we ever do that here so I avoided providing an override of that function to cover this.